pkgs/applications/misc/clifm/default.nix

```diff
@@ -0,0 +1,59 @@
+{ stdenv, lib, fetchFromGitHub, makeWrapper, libcap, acl, file, readline
+, withRemoteSupport ? true, sshfs, curlftpfs, cifs-utils
+, withCompressionSupport ? true, atool, archivemount, cdrkit, p7zip, cdrtools
+, withIconSupport ? false, icons-in-terminal ? null
```

Comment:
```suggestion
, withIconSupport ? false, icons-in-terminal
```

Reply:
icons-in-terminal is a shell script and it is not on nixpkgs. However, like other extensions, it can be wrapped by user, so I remove it from dependencies.